app/controllers/merge_requests_controller.rb

```diff
@@ -11,7 +11,7 @@ def helpdesk_ticket; end
 
   def create
     ActiveRecord::Base.transaction do
-      @merge_request = MergeRequest.create!(merge_request_params.merge(status: :incomplete))
+      @merge_request = MergeRequest.create!(merge_request_params.merge(status: :incomplete, requester_id: current_user.id))
```

Comment:
Out of the scope of this ticket but I added the requester_id when creating a ticket